src/status_im2/subs/communities.cljs

```diff
@@ -150,6 +151,19 @@
     (get communities identity)
     counts)))
 
+(re-frame/reg-sub
+ :communities/my-pending-requests-to-join
+ (fn [[]]
+   [(re-frame/subscribe [:my-pending-requests-to-join])])
```

Comment:
This one is weird @J-Son89. Could it be a leftover? Can't you just use the arrow `:<-` syntax sugar? The syntax you used here is often used when there's a parameter for the subscription, but there's none.

Reply:
yeah, I'll try update the nicer version 👍 . Should be fine afaik